src/main/java/com/mraof/minestuck/network/computer/BurnDiskPacket.java

```diff
@@ -0,0 +1,76 @@
+package com.mraof.minestuck.network.computer;
+
+import com.mraof.minestuck.blockentity.ComputerBlockEntity;
+import com.mraof.minestuck.computer.ProgramData;
+import com.mraof.minestuck.network.PlayToServerPacket;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+
+import java.util.Random;
+
+public class BurnDiskPacket implements PlayToServerPacket
+{
+	private final BlockPos bePos;
+	private final int programId;
+	
+	public BurnDiskPacket(BlockPos pos, int programId)
+	{
+		this.bePos = pos;
+		this.programId = programId;
+	}
+	
+	public static BurnDiskPacket create(ComputerBlockEntity be, int programId)
+	{
+		return new BurnDiskPacket(be.getBlockPos(), programId);
+	}
+	
+	@Override
+	public void encode(FriendlyByteBuf buffer)
+	{
+		buffer.writeBlockPos(bePos);
+		buffer.writeInt(programId);
+	}
+	
+	public static BurnDiskPacket decode(FriendlyByteBuf buffer)
+	{
+		BlockPos bePos = buffer.readBlockPos();
+		int programId = buffer.readInt();
+		return new BurnDiskPacket(bePos, programId);
+	}
+	
+	@Override
+	public void execute(ServerPlayer player)
+	{
+		Level level = player.level;
+		BlockEntity blockEntity = level.getBlockEntity(bePos);
```

Comment:
There needs to be some safety against triggering chunk loading, and against interacting with computers far away. (See use of `ComputerBlockEntity.forNetworkIfPresent()` in other computer-related packages)

Reply:
I believe I fixed it